src/js/models/layer.js

```diff
@@ -484,32 +594,37 @@ define(function (require, exports, module) {
      */
     Layer.fromGroupDescriptor = function (documentID, layerID, layerName, isGroupEnd,
         isArtboard, boundsDescriptor) {
-        return new Layer({
-            id: layerID,
-            key: documentID + "." + layerID,
-            name: isGroupEnd ? "</Layer Group>" : layerName,
-            kind: isGroupEnd ? layerLib.layerKinds.GROUPEND : layerLib.layerKinds.GROUP,
-            visible: true,
-            locked: false,
-            expanded: true,
-            isBackground: false,
-            opacity: 100,
-            selected: true, // We'll set selected after moving layers
-            fill: null,
-            stroke: null,
-            dropShadows: Immutable.List(),
-            innerShadows: Immutable.List(),
-            effects: LayerEffect.EMPTY_EFFECTS,
-            mode: "passThrough",
-            proportionalScaling: false,
-            isArtboard: !!isArtboard,
-            bounds: isArtboard ? new Bounds(boundsDescriptor) : null,
-            isLinked: false,
-            vectorMaskEnabled: false,
-            vectorMaskEmpty: true,
-            textWarningLevel: 0,
-            initialized: true
-        });
+        var Bounds = require("./bounds"),
```

Comment:
I'll give you the benefit of the doubt, is there a reason these are in function scope?